org.openshot.OpenShot.json

```diff
@@ -0,0 +1,385 @@
+{
+    "app-id": "org.openshot.OpenShot",
+    "branch": "stable",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.9",
+    "sdk": "org.kde.Sdk",
+    "command": "openshot-qt",
+    "writable-sdk": true,
```

Comment:
Can't have a writable sdk.

Reply:
If you have an idea how I can tell qmake to search for Qt modules in both /usr and /app, then this would not be needed. And does it even make a difference after the app is built?

Reply:
...and this would not be the first app:

https://github.com/flathub/org.freedesktop.Bustle/blob/master/org.freedesktop.Bustle.json

Reply:
Does this help? https://github.com/flathub/io.github.cloose.CuteMarkEd/blob/master/io.github.cloose.CuteMarkEd.json#L76

Reply:
I will try that...

Reply:
It helped, thanks!